pkg/services/grafana-apiserver/config.go

```diff
@@ -47,7 +47,7 @@ func newConfig(cfg *setting.Cfg, features featuremgmt.FeatureToggles) *config {
 
 	return &config{
 		enabled:     features.IsEnabledGlobally(featuremgmt.FlagGrafanaAPIServer),
-		devMode:     cfg.Env == setting.Dev,
+		devMode:     features.IsEnabledGlobally(featuremgmt.FlagGrafanaAPIServerEnsureKubectlAccess),
```

Comment:
@toddtreece @grafana/grafana-app-platform-squad  -- this changes the behavior for when we auto-start the extra kubectl port.

This now requires a feature flag (and dev mode) rather than just dev mode.  This change was required to simplify the integration test setup, but I think is generally better anyway.

Reply:
the `README` probably also needs to be updated with this info for kubectl access